sbt-scalajs-bundler/src/main/scala/scalajsbundler/WebpackDevServer.scala

```diff
@@ -13,22 +13,19 @@ private [scalajsbundler] class WebpackDevServer {
   /**
     * @param workDir - path to working directory for webpack-dev-server
     * @param configPath - path to webpack config.
-    * @param port - port, on which the server will operate.
     * @param extraArgs - additional arguments for webpack-dev-server.
     * @param logger - a logger to use for output
     */
   def start(
     workDir: File,
     configPath: File,
-    port: Int,
```

Comment:
Is it expected to configure the port via webpack config?

Reply:
Yes as the invocation method the the dev-server is different with webpack 5 (I mentioned it as a possible breaking change in the issue description)